zproject/urls.py

```diff
@@ -746,6 +747,11 @@
     path("api/v1/events/internal", get_events_internal),
 ]
 
+# tusd views
+urls += [
+    rest_path("tusd/hooks", POST=(handle_tusd_hook, {"override_api_url_scheme"})),
+]
```

Comment:
What's the reasoning for why we're using `override_api_url_scheme` for these endpoints?